terraform/gcp/pipeline/secure/target_bigquery.tf

```diff
@@ -0,0 +1,68 @@
+module "bq_bad_rows_topic" {
```

Comment:
Weren’t we getting rid of these?

Reply:
You mean secure or the bad_rows_topic?  We need the bad_rows_topic.

Reply:
Secure

Reply:
Can strip it out in this release if you like or can do that as a separate action?  My thinking was to first strip it out of the docs and references to it and potentially just leave it as a reference here?